corehq/apps/app_manager/templates/app_manager/partials/report_configs_js.html

```diff
@@ -0,0 +1,58 @@
+{% load hq_shared_tags %}
+
+{% include "app_manager/partials/nav_menu_media_js_common.html" %}
+{% include "app_manager/partials/nav_menu_media_js.html" with item=multimedia.menu %}
+
+<script>
+    $(function() {
+        var saveURL = "{% url "corehq.apps.app_manager.views.edit_report_module" domain app.id module.id %}";
+        var reportModule = new ReportModule({
+            moduleName: {{ module.name|JSON }},
+            moduleFilter: "{{ module.module_filter|escapejs }}",
+            availableReports: {{ all_reports|JSON }}, // structure for all reports
+            currentReports: {{ current_reports|JSON }}, // config data for app reports
+            saveURL: saveURL,
+            lang: {{ lang|JSON }},
+            menuImage: menuImage,
+            menuAudio: menuAudio
+        });
+        ko.applyBindings(reportModule, $("#save-button")[0]);
```

Comment:
super picky, but you could just use `document.getElementById` instead of going through jQuery